torch/_inductor/ir.py

```diff
@@ -781,13 +781,10 @@ def outer_reduction_splits(reduction_numel_hint, numel_hint):
                 # No need to split.
                 return ReductionHint.INNER, split
             if (
-                len(ranges) == 0
+                all(r == 1 for r in ranges)
```

Comment:
Please add a comment to explain, e.g. it only works for tensor-level reduction.

Reply:
In fact, all this code is within a conditional for `numel_hint == 1`, hence this check will always be true, I can just remove it.